Tests/FormworksTests/ViewModels/FWFormViewModelTests.swift

```diff
@@ -12,7 +12,7 @@ final class FWFormViewModelTests: XCTestCase {
     var sut: FWFormViewModel!
 
     func testProperties() {
-        let configuration = FWConfiguration(json: TestFixtures.formData, style: .light)
+        let configuration = FWConfiguration(json: TestFixtures.formData, style: .custom(FWStyleSpecification()))
```

Comment:
I think that we maybe need to specify in the docs that the user needs to put ```FWStyleSpecification()``` just like here in the test, so in that way he won't be confused.